mw-webapp/src/dataAccessLogic/useGetDataTable.ts

```diff
@@ -0,0 +1,22 @@
+import {useEffect, useState} from "react";
+import {getDayReports} from "src/dataAccessLogic/getDayReports";
+import {DayReport} from "src/model/businessModel/DayReport";
+
+/**
+ * Reports requests: {@link getDayReports}
+ * @returns {DayReport[]} data of reports
+ */
+export const useGetDataTable = () => {
```

Comment:
1. DAL should not be connected to requests, check comment
2.  why do you mention getDayReports in comment, why son;t you want to encapsulate it
3. `useGetDataTable` -  looks like universal hook for all tables, but it is not true. Specify naming

Reply:
1. Done
2. Done
3. Done